src/cueButton.vue

```diff
@@ -68,6 +71,14 @@ module.exports = {
 				}
 			});
 			return active;
+		},
+		cueMethod: function() {
```

Comment:
Why is this method needed? should just be able to access the method via `this.method`

Reply:
It's obsolete now with the switch function. It's been removed.